README.md

```diff
@@ -266,6 +266,19 @@ Example:
 }
 ```
 
+###### `dataTransform` (Function | async Function)
+
+Optional. Relevant only when using dynamic (js) config.
+A function to allow manipulation on the data before making the request.
+
+Here is an example for adding a new field named `wiki` to the data:
+```
+{
+  ...
+  "dataTransform": items => items.map(item => Object.assign(item, { wiki: `https://en.wikipedia.org/wiki/${item.name}` }))
```

Comment:
I don't there's necessarily an iterable object here. The `dataTransform` method in `POST` and `PUT` should get the `body` of the request and it should be able to manipulate it.